src/platform/testing/e2e/cypress/support/commands/expandAccordions.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Expands all accordions and AdditionalInfo components.
+ */
+Cypress.Commands.add('expandAccordions', () => {
+  Cypress.log();
```

Comment:
Will this log an empty string?

Reply:
This logs out `EXPANDACCORDIONS` in the list of steps.